test/dotnet-watch.Tests/HotReload/ApplyDeltaTests.cs

```diff
@@ -120,5 +120,56 @@ public async Task BlazorWasm()
             //UpdateSourceFile(Path.Combine(testAsset.Path, "Pages", "Index.razor"), newSource);
             //await App.AssertOutputLineStartsWith(MessageDescriptor.HotReloadSucceeded);
         }
+
+        // Test is timing out on .NET Framework: https://github.com/dotnet/sdk/issues/41669
```

Comment:
Is the test still timing out? Can you try to uncomment so that we can look at logs?

Reply:
Will try.

Reply:
Looks like it's working! @tmat do I need to backport those changes to https://github.com/dotnet/sdk/pull/40725, or will there be an automatic PR somehow from this current one?

Reply:
Also, the build `Build AoT: macOS (x64)` has timed out, I don't think I can restart it by myself.